staff/models/batch.py

```diff
@@ -64,6 +65,19 @@ def fully_enrolled(self):
             return True
         return False
 
+    def html_formatted_batch_price(self):
+        base_price = settings.CODING_BASICS_REGISTRATION_FEE_SGD
+        html_formatted_price = f"<span class='float-end d-none d-xl-block'>${base_price}</span>"
+        html_formatted_price += f"<div class='lh-lg d-xl-none my-10'>${base_price}<div>"
+        if self.weeks_to_start() >= 2:
```

Comment:
please move this logic to the view! perhaps the caller can do the computation, then pass in `discounted_price` and `original_price`. reasons for this separation is:

1. this method name implies we are only formatting the batch price into html. it does not say anything about handling conditional pricing.
2. generally each method should do 1 thing.